_config.yml

```diff
@@ -25,7 +25,7 @@ baseurl: "/bagel-box"
 # the base hostname & protocol for your site, e.g. http://example.com
 url: "https://serviceinnovationlab.github.io/"
 
-github_username: ServiceInnovationLab
+github_username: Service Innovation Lab GitHub
```

Comment:
Why this change?

Reply:
You could just remove it from the config and change it in the footer

Reply:
The design 
![image](https://user-images.githubusercontent.com/13788498/69924456-5781b600-1510-11ea-84eb-31ef741e7590.png)